infrastructure/src/main/resources/logback.prod.xml

```diff
@@ -31,7 +31,7 @@
         </triggeringPolicy>
         <encoder>
             <pattern>
-                %date{yyyy-MM-dd'T'HH:mm:ss.SSSXXX, UTC} | %msg%n%xException
+                %msg%n%xException
```

Comment:
This will make it hard to inspect the metrics files manually. I suggest you just post process the logs and truncate these timestamps.